readme.txt

```diff
@@ -40,13 +32,13 @@ Currently the plugin includes the following performance modules:
 = After activation =
 
 1. Visit the new **Settings > Performance** menu.
-2. Enable the individual modules you would like to use.
+2. Enable the individual plugins you would like to use.
 
 == Frequently Asked Questions ==
 
 = What is the purpose of this plugin? =
 
-The primary purpose of the Performance Lab plugin is to allow testing of various performance modules for which the goal is to eventually land in WordPress core. It is essentially a collection of "feature plugins", which makes it different from other performance plugins that offer performance features which are not targeted at WordPress core and potentially rely on functionality that would not be feasible to use in WordPress core. The list of available modules will regularly change: Existing modules may be removed after they have been released in WordPress core, while new modules may be added in any release.
+The primary purpose of the Performance Lab plugin is to allow testing of various performance plugins for which the goal is to eventually land in WordPress core. It is essentially a collection of "feature plugins", which makes it different from other performance plugins that offer performance features which are not targeted at WordPress core and potentially rely on functionality that would not be feasible to use in WordPress core. The list of available plugins will regularly change: Existing plugins may be removed after they have been released in WordPress core, while new modules may be added in any release.
```

Comment:
> Existing plugins may be removed after they have been released in WordPress core, while new modules may be added in any release.

I think modules is confusing here given we are removing them completely. Should we term it as standalone or feature plugin here?